Formula/blocky.rb

```diff
@@ -0,0 +1,36 @@
+class Blocky < Formula
+  desc "Fast and lightweight DNS proxy as ad-blocker for local network"
+  homepage "https://0xerr0r.github.io/blocky"
+  url "https://github.com/0xerr0r/blocky/archive/refs/tags/v0.21.tar.gz"
+  sha256 "070babaf736495c9d4fa6af37f3d55501572939fff92fad4ec53340b579357da"
+  license "Apache-2.0"
+  head "https://github.com/0xerr0r/blocky.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/0xERR0R/blocky/util.Version=#{version}
+      -X github.com/0xERR0R/blocky/util.BuildTime=#{time.iso8601}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags)
+    sbin.install_symlink bin/"blocky"
```

Comment:
Better do 
```suggestion
    system "go", "build", *std_go_args(ldflags: ldflags, output: sbin/"blocky")
```

Reply:
Thanks!

Reply:
Do we have to update the binary path for the client test?

Reply:
The binary path for the client test has been changed to sbin.